### PR TITLE
Wait on uploading user creds

### PR DESCRIPTION
**Pull request by yebrahim**
Wait until the upload is complete and send an ack message back so the notebook editor can wait on the upload.

### CODE REVIEW

File: sources/web/datalab/polymer/notebook.ts [hunk 1]
```diff
@@ -44,7 +44,13 @@ async function processMessageEvent(e: MessageEvent) {
...
 
   if (message.command === CommandId.UPLOAD_USER_CREDS) {
-    ApiManagerFactory.getInstance().uploadOauthAccessToken();
+    try {
+      await ApiManagerFactory.getInstance().uploadOauthAccessToken();
+    } catch (e) {
+      message.command = CommandId.ERROR;
+      message.arguments = e;
+    }
+    sendMessageToNotebookEditor(message);
```
Comment (chmeyers) [inline sources/web/datalab/polymer/notebook.ts]: Should this be inside the catch?

Comment (yebrahim) [reply sources/web/datalab/polymer/notebook.ts]: We want to send an ack always, the catch is just changing the command to error and sending the error along.

Comment (jimmc) [reply sources/web/datalab/polymer/notebook.ts]: You are essentially reusing 'message' for a completely different purpose. I think the intent of the code would be more clear if you did this:
try {
  await ApiManagerFactory...
  sendMessageToNotebookEditor(message);
} catch (e) {
  const errorMessage = {
    command: CommandId.ERROR;
    arguments: e
  };
  sendMessageToNotebookEditor(errorMessage);
  return;
}

Comment (yebrahim) [reply sources/web/datalab/polymer/notebook.ts]: Done. PTAL.

**Pull request merged**